iam-password-policy/change-password-policy.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+
+from copy import Error
+import boto3
+from botocore.exceptions import ClientError
+import os
+import logging
+from botocore.session import Session
+
+
+
+
+
+def main(args, logger):
+    '''Executes the Primary Logic of the Fast Fix'''
+
+    # If they specify a profile use it. Otherwise do the normal thing
+    if args.profile:
+        session = boto3.Session(profile_name=args.profile)
+    else:
+        session = boto3.Session()
+
+    try:
+        response = setPasswordPolicy(args, client= session.client(service_name='iam'))
+        logging.debug("Response:"+str(response))
```

Comment:
You instantiate logger on 156, then pass it as a variable to main, but here you're using the logging class itself rather than the logger instance.

Reply:
Fixed.